org.eclipse.scout.rt.platform.test/src/main/java/org/eclipse/scout/rt/testing/platform/runner/JUnitExceptionHandler.java

```diff
@@ -73,4 +76,20 @@ public void throwOnError() throws Throwable {
     m_errors.clear();
     throw throwable;
   }
+
+  /**
+   * Ignores a specified {@link Throwable} (or a subclass of it) once regarding this exception handler exception memory.
+   * This method only removes an instance of the specified {@link Throwable} once from the error list, see
+   * {@link #getErrors()}. If {@link Throwable} is thrown by the specified {@link IRunnable} it is not catched by this
+   * method.
+   */
+  public void ignoreExceptionOnce(Class<? extends Throwable> throwableToBeIgnored, IRunnable runnable) throws Exception {
+    ArrayList<Throwable> previousErrors = new ArrayList<>(m_errors);
+    try {
+      runnable.run();
+    }
+    finally {
+      m_errors.stream().filter(throwableToBeIgnored::isInstance).filter(not(previousErrors::contains)).findFirst().ifPresent(m_errors::remove);
```

Comment:
Suggestion: Write stream operations on multiple lines to increase readability:

```
      m_errors.stream()
          .filter(throwableToBeIgnored::isInstance)
          .filter(not(previousErrors::contains))
          .findFirst()
          .ifPresent(m_errors::remove);
```

Reply:
Thanks, changed as suggested in 1b8b76fe6bbf68b9c423691bc7e8b4f11e7fd220